test_scripts/Defects/5_1/1886_OnVehicleData_PoliciesManager_does_not_allow_all_requested_params_in_case_parameters_field_is_omited.lua

```diff
@@ -0,0 +1,191 @@
+local common = require("user_modules/sequences/actions")
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Functions ]]
+local function ptuUpdateFuncDissalowedRPC(tbl)
+  local VDgroup = {
+    rpcs = {
+      GetVehicleData = {
+        hmi_levels = {"BACKGROUND", "FULL", "LIMITED"},
+        -- parameters omitted
+      },
+      OnVehicleData = {
+        hmi_levels = {"BACKGROUND", "FULL", "LIMITED"},
+        -- parameters omitted
+      },
+      SubscribeVehicleData = {
+        hmi_levels = {"BACKGROUND", "FULL", "LIMITED"},
+        -- parameters omitted
+      },
+      UnsubscribeVehicleData = {
+        hmi_levels = {"BACKGROUND", "FULL", "LIMITED"},
+        -- parameters omitted
+      },
+      SendLocation = {
+        hmi_levels = {"BACKGROUND", "FULL", "LIMITED"},
+        -- parameters omitted
+      }
+    }
+  }
+  tbl.policy_table.functional_groupings["NewTestCaseGroup"] = VDgroup
+  if tbl.policy_table.functional_groupings["SendLocation"] then
+    tbl.policy_table.functional_groupings["SendLocation"] = nil
+  end
+  tbl.policy_table.app_policies[config.application1.registerAppInterfaceParams.fullAppID].groups = {"Base-4", "NewTestCaseGroup"}
+end
+
+local gpsResponse = {
+  longitudeDegrees = -180,
+  latitudeDegrees = 90,
+  utcYear = 2100,
+  utcMonth = 12,
+  utcDay = 22,
+  utcHours = 20,
+  utcMinutes = 50,
+  utcSeconds = 50,
+  compassDirection = "NORTH",
+  pdop = 1000,
+  hdop = 1000,
+  vdop = 1000,
+  actual = true,
+  satellites = 31,
+  dimension = "2D",
+  altitude = 10000,
+  heading = 359.99,
+  speed = 500,
+  shifted = true
+}
+
+local gpsDataResponse = {
+  longitudeDegrees = 100,
+  latitudeDegrees = 20,
+  utcYear = 2050,
+  utcMonth = 10,
+  utcDay = 30,
+  utcHours = 20,
+  utcMinutes = 50,
+  utcSeconds = 50,
+  compassDirection = "NORTH",
+  pdop = 5,
+  hdop = 5,
+  vdop = 5,
+  actual = false,
+  satellites = 30,
+  dimension = "2D",
+  altitude = 9500,
+  heading = 350,
+  speed = 450,
+  shifted = true
+}
+
+local requestParams = {
+  longitudeDegrees = 1.1,
+  latitudeDegrees = 1.1,
+  addressLines =
+  { "line1", "line2" },
+  address = {
+    countryName = "countryName",
+    countryCode = "countryName",
+    postalCode = "postalCode",
+    administrativeArea = "administrativeArea",
+    subAdministrativeArea = "subAdministrativeArea",
+    locality = "locality",
+    subLocality = "subLocality",
+    thoroughfare = "thoroughfare",
+    subThoroughfare = "subThoroughfare"
+  },
+  timeStamp = {
+    millisecond = 0,
+    second = 40,
+    minute = 30,
+    hour = 14,
+    day = 25,
+    month = 5,
+    year = 2017,
+    tz_hour = 5,
+    tz_minute = 30
+  },
+  locationName = "location Name",
+  locationDescription = "location Description",
+  phoneNumber = "phone Number",
+  deliveryMode = "PROMPT",
+}
+
+local function GetVD()
+  local cid = common.getMobileSession():SendRPC("GetVehicleData", {gps = true})
```

Comment:
Script should check processing of all VD parameters not only of `gps`, find the example in https://github.com/smartdevicelink/sdl_atf_test_scripts/blob/develop/test_scripts/API/VehicleData/GetVehicleData/001_Success_flow.lua

Reply:
@GetmanetsIrina  You can find the updates in the commit 20149d4